localization/i18n/de/OrcaSlicer_de.po

```diff
@@ -8123,7 +8126,7 @@ msgid "Gizmo cut"
 msgstr "Trennen"
 
 msgid "Gizmo Place face on bed"
-msgstr "Fläche auf Druckbett platzieren "
+msgstr "Gizmo-Fläche auf Druckbett platzieren "
```

Comment:
According to all the other "Gizmo" translations ("Bewegen", "Skalieren" ...)
This should say as follows, otherwise it doesn't make sense:
```suggestion
msgstr "Fläche auf Druckbett platzieren"
```

Reply:
I think there are several other mentions of Gizmo in the translation. That's why I wanted to make it consistent. It's not a major thingy.